File: JavaScript Front End/src/Messages/CommandResponseMessages/GraphBuildingTaskCreationFailureResponseMsg.js
const GraphBuildingFailureResponseMsg = require('./GraphBuildingFailureResponseMsg.js').GraphBuildingFailureResponseMsg;

class GraphBuildingTaskCreationFailureResponseMsg extends GraphBuildingFailureResponseMsg
{
    constructor(jsonMessage)
    {
        super(jsonMessage);
        this.createdNodeTemporaryId = this.jsonMessage["createdEdgeTemporaryId"];
    }
}

module.exports = {GraphBuildingTaskCreationFailureResponseMsg};

File: Java Back End/src/InfoSecCooker/Communication/Messages/CommandResponseMessages/GraphBuildingTaskCreationFailureResponseMsg.java
package InfoSecCooker.Communication.Messages.CommandResponseMessages;

import InfoSecCooker.GraphLogic.GraphBuilding.Exceptions.GraphBuildingException;

public class GraphBuildingTaskCreationFailureResponseMsg extends GraphBuildingFailureResponseMsg
{
    private long createdNodeTemporaryId;

    public GraphBuildingTaskCreationFailureResponseMsg(GraphBuildingException exception, long createdNodeTemporaryId, String additionalInfo)
    {
        super(exception, additionalInfo);
        this.createdNodeTemporaryId = createdNodeTemporaryId;
    }

    public GraphBuildingTaskCreationFailureResponseMsg(GraphBuildingException exception, long createdNodeTemporaryId)
    {
        super(exception, "");
        this.createdNodeTemporaryId = createdNodeTemporaryId;
    }
}
